ui/src/logs/components/LogViewerChart.tsx

```diff
@@ -0,0 +1,33 @@
+import React, {PureComponent} from 'react'
+import LineGraph from 'src/shared/components/LineGraph'
+import {DEFAULT_LINE_COLORS} from 'src/shared/constants/graphColorPalettes'
+
+import {TimeRange} from 'src/types'
+
+interface Props {
+  onZoom: (lower: string, upper: string) => void
+  timeRange: TimeRange
+  data: object[]
+}
+
+class LogViewerChart extends PureComponent<Props> {
+  public render() {
+    const {timeRange, data, onZoom} = this.props
+    return (
+      <LineGraph
+        onZoom={onZoom}
+        queries={[]}
+        data={data}
+        displayOptions={{animatedZooms: false}}
+        setResolution={this.setResolution}
+        isBarGraph={true}
+        timeRange={timeRange}
+        colors={DEFAULT_LINE_COLORS}
+      />
+    )
+  }
+
+  private setResolution = () => {}
```

Comment:
i know you mentioned this was rough, so just wanted to check on this being what is intended currently? :)